index.d.ts

```diff
@@ -0,0 +1,53 @@
+declare namespace camelcaseKeys {
+	interface Options {
+		/**
+		Recurse nested objects and objects in arrays.
+
+		@default false
+		*/
+		readonly deep?: boolean;
+
+		/**
+		Exclude keys from being camelCased.
+
+		@default []
+		*/
+		readonly exclude?: (string | RegExp)[];
```

Comment:
This is inconsistent with how we have it in most of the other definitions. We should pick a style and document it in the style guide. Either `Array<string | RegExp>` or `(string | RegExp)[]`. Any preference? 


I know we have:

> Use the array shorthand type notation; number[], not Array<number>.

But it's not clear enough that applies when having multiple types in the array.

Reply:
My personal preference is the same as the one in the DT styleguide:

- simple types: `string[]`, `number[]`
- complex types: `Array<string | RegExp>`

I was using the array shorthands because I interpreted your styleguide that it always requires them.

Reply:
Alright. That's what I prefer too. Let's update the styleguide to be clearer on this.